database/migrations/2025_10_01_174110_clinics_doctors.php

```diff
@@ -14,7 +14,9 @@ public function up(): void
             $table->id();
             $table->foreignId('clinic_id')->constrained('clinics', 'id')->cascadeOnDelete();
             $table->foreignId('doctor_id')->constrained('doctors', 'id')->cascadeOnDelete();
```

Comment:
Not needed since you're following Laravel conventions for naming

```suggestion
$table->foreignId('clinic_id')->constrained()->cascadeOnDelete();
$table->foreignId('doctor_id')->constrained()->cascadeOnDelete();
```